docs/content/integrations/import.md

```diff
@@ -332,7 +332,24 @@ Use the full XML export template from Nexpose.
 Nikto
 -----
 
-XML output
+Nikto web server scanner - https://cirt.net/Nikto2
+
+XML output (old and new nxvmlversion=\"1.2\" type) or JSON output
+See: https://github.com/sullo/nikto
+
+To launch a simple run if you have Docker installed:
```

Comment:
Would we want maybe to re-direct to the official tool's pages, so that we don't have to maintain anything here?

Reply:
Honestly, Nikto seems to not be very well (only one maintainer with few commits since 6 months), I'm afraid that this doc will disappear.
I can remove this part if you want.

Reply:
I just don't want for us to set the wrong expectations that we will do that for every parser, because we likely won't :-)

Reply:
ok. you convinced me. let's simplify that.